src/Umbraco.Cms.Api.Management/Controllers/DocumentVersion/AllDocumentVersionController.cs

```diff
@@ -31,6 +31,8 @@ public AllDocumentVersionController(
     [ProducesResponseType(typeof(PagedViewModel<DocumentVersionItemResponseModel>), StatusCodes.Status200OK)]
     [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
     [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+    [EndpointSummary("Gets a paged result of all versions for a specific document")]
+    [EndpointDescription("A documentId is required. Culture, skip, and take parameters are optional. Returned is data relevant to that document, e.g., document type, user ID, version date, and whether it is the current published or draft version. This is useful when you need to find a document version ID for other queries.")]
```

Comment:
```suggestion
    [EndpointDescription("Gets a paginated collection of versions for a specific document and optional culture. Each result describes the version and includes details of the document type, editor, version date, and published status.")]
```

Simplified a little to reduce the effort moving forward and remove things that are already available in the Swagger UI.  Also removed the "this is useful when..." part.  Not sure about that but again it may be a bit burdensome if it presents a pattern for the rest of the APIs and it's perhaps hard to really think about why people might want a given result.